mono/mini/exceptions-x86.c

```diff
@@ -664,7 +664,7 @@ get_throw_trampoline (const char *name, gboolean rethrow, gboolean llvm, gboolea
 		// ebx contains the got address.
 		// So emit the got address loading code too
 		code = mono_arch_emit_load_got_addr (start, code, NULL, &ji);
-		code = mono_arch_emit_load_aotconst (start, code, &ji, MONO_PATCH_INFO_JIT_ICALL_ADDR, corlib ? MONO_JIT_ICALL_mono_x86_throw_corlib_exception : MONO_JIT_ICALL_mono_x86_throw_exception);
+		code = mono_arch_emit_load_aotconst (start, code, &ji, MONO_PATCH_INFO_JIT_ICALL_ADDR, corlib ? GUINT_TO_POINTER (MONO_JIT_ICALL_mono_x86_throw_corlib_exception) : GUINT_TO_POINTER (MONO_JIT_ICALL_mono_x86_throw_exception));
```

Comment:
Put the cast on the outside to avoid repeating it.
The ? and : are of the same type.

`GUINT_TO_POINTER (corlib ? MONO_JIT_ICALL_mono_x86_throw_corlib_exception : MONO_JIT_ICALL_mono_x86_throw_exception);`